src/vacuum-card.js

```diff
@@ -104,6 +105,31 @@ class VacuumCard extends LitElement {
     this.callService('set_fan_speed', { fan_speed });
   }
 
+  getStatusValues() {
+    this.type =
+      typeof this.entity.attributes.valetudo_state === 'undefined'
+        ? 'r5_max'
```

Comment:
I don't like we need to specify model manually. I believe this should be as generic as possible.
We can just check if, for example, `cleaned_area ` has any value, if not, then fallback to `cleanArea ` and so one.

Reply:
Yeah, sure, can remove the model type..
Were you thinking to check each variable and fall back? From what I know, valetudo user the same attributes independent of model.

Reply:
I've almost finished it myself. Wait for a fix.